x-pack/plugins/infra/public/pages/logs/log_entry_categories/page_content.tsx

```diff
@@ -39,12 +40,23 @@ export const LogEntryCategoriesPageContent = () => {
 
   const { fetchJobStatus, setupStatus, jobStatus } = useLogEntryCategoriesModuleContext();
 
+  const [isFlyoutOpen, setIsFlyoutOpen] = useState<boolean>(false);
+  const openFlyout = useCallback(() => setIsFlyoutOpen(true), []);
+  const closeFlyout = useCallback(() => setIsFlyoutOpen(false), []);
+
   useEffect(() => {
     if (hasLogAnalysisReadCapabilities) {
       fetchJobStatus();
     }
   }, [fetchJobStatus, hasLogAnalysisReadCapabilities]);
 
+  // Open flyout if there are no ML jobs
+  useEffect(() => {
+    if (jobStatus['log-entry-categories-count'] === 'missing') {
```

Comment:
Could

```suggestion
    if (setupStatus.type === 'required' && setupStatus.reason === 'missing') {
```

 be better suited for this?

Reply:
I honestly don't know 🤷‍♂️. Do you see any downsides with the current check?

Reply:
The ML module could grow to contain multiple jobs. The purpose of the setup status is to abstract that away and present an overall state so the check wouldn't have to change once a new job is added to the ML module.